selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/platform/social/SpaceSettingManagement.java

```diff
@@ -11,6 +12,9 @@
 
 import static com.codeborne.selenide.Selectors.*;
 import static com.codeborne.selenide.Selenide.$;
+import static com.sun.deploy.cache.Cache.exists;
```

Comment:
Is this method used ?

Reply:
No i will remove unused imports , Fixed done.